src/index.tsx

```diff
@@ -53,4 +55,12 @@ if (process.env.NODE_ENV !== 'production') {
   axe(React, ReactDOM, 1000, config);
 }
 
+if (process.env.PREVIEW === 'true') {
```

Comment:
I think `process.env.NODE_ENV` is commonly used for the same sort of thing. Usual values for it are `development` and `production`, but nothing stops us from using a `preview` value. Maybe that's worth exploring? It would just be a little more conventional, I don't expect it to have any real material impact.

Reply:
oh yes my thought originally was that we would want to be able to have preview (i.e mirage js) in hot-reload dev server and "production" build (i.e. for PR preview or main preview or upstream cryostatio).

So i use a different environment var to control (i.e. set in webpack). What do u think?

Reply:
Tho, from reading this: https://webpack.js.org/guides/production/#specify-the-mode

I think mode: `production` or `NODE_ENV: production` is needed for some tools to work. So, its not really recommended to set it to `preview`?

Reply:
Latest updates only uses `process.env.PREVIEW` within webpack process to apply mirage bundle. So, no checking in the actual bundle.